Gopkg.toml

```diff
@@ -21,4 +21,4 @@ required = [
 # to prevent unexpected downgrades; see https://github.com/percona/exporter_shared/releases/tag/v0.6.0
 [[constraint]]
   name = "github.com/percona/exporter_shared"
-  version = "0.6.0"
+  version = "0.7.2"
```

Comment:
Why did you update exporter_shared version?

Reply:
It has transitive dependency on `github.com/prometheus/common`. So firstly I updated `prometheus/common` in `exporter_shared` and then I updated `exporter_shared` in `mongodb_exporter`.